spec/delayed_job_chainable_hooks_spec.rb

```diff
@@ -1,5 +1,10 @@
+# frozen_string_literal: true
+
 describe DelayedJobChainableHooks do
   before do
+    spec_dir = File.dirname(__FILE__)
+    require "#{spec_dir}/fixtures/job_classes"
```

Comment:
Should we require the file from `spec_helper` so we get more consistent constant loading across tests? At a minimum this doesn't need to be done in a `before` block and could utilize `require_relative`.

Reply:
Sure. I'd put it in spec_helper.rb first and then moved it here for context, but I was on the fence.